fetch.bs

```diff
@@ -301,6 +301,9 @@ following <a for=struct>items</a>: [[RESOURCE-TIMING]] [[NAVIGATION-TIMING]]
  <dt><dfn export for="response body info" id="fetch-timing-info-decoded-body-size">decoded
  size</dfn> (default 0)
  <dd>A number.
+ <dt><dfn export for="response body info" id="fetch-timing-info-response-status">response
```

Comment:
This is not needed. You have [response's status](https://fetch.spec.whatwg.org/#concept-response-status) when you set the `report timing steps` in [fetch response handover](https://fetch.spec.whatwg.org/#fetch-finale). I would add it as another parameter to `Mark resource timing`.

Also, as we talked on the Chromium CL, I don't think we should automatically make this TAO-protected rather than CORS-protected. It's not exactly timing information.

Reply:
Okay, just to be sure I understand. We would pass in the status as a new parameter to [Mark resource time](https://w3c.github.io/resource-timing/#dfn-mark-resource-timing) which would in turn pass it to [Setup the resource timing entry](https://w3c.github.io/resource-timing/#dfn-setup-the-resource-timing-entry) where we would use it to set the entry's `responseStatus`.

> Also, as we talked on the Chromium CL, I don't think we should automatically make this TAO-protected rather than CORS-protected. It's not exactly timing information.

Yeah, we don't want to. I am a bit unsure as to how we would would specify this in the specification . I initially thought mentioning in the Note in [Cross-origin Resources section of resource timing](https://w3c.github.io/resource-timing/#sec-cross-origin-resources) that the status would be `0` if `CORS check` fails for the cross origin resource would suffice, but it seems I'm missing something.

Reply:
> Okay, just to be sure I understand. We would pass in the status as a new parameter to [Mark resource time](https://w3c.github.io/resource-timing/#dfn-mark-resource-timing) which would in turn pass it to [Setup the resource timing entry](https://w3c.github.io/resource-timing/#dfn-setup-the-resource-timing-entry) where we would use it to set the entry's `responseStatus`.

Exactly

> 
> > Also, as we talked on the Chromium CL, I don't think we should automatically make this TAO-protected rather than CORS-protected. It's not exactly timing information.
> 
> Yeah, we don't want to. I am a bit unsure as to where the exact place to specify this is. Does it make sense to add this to [fetch response handover](https://fetch.spec.whatwg.org/#fetch-finale) as well - If the CORS check fails pass in `0` as the status to `Mark resource timing`

Actually I don't think you need to do anything. If this is a no-cors request, `response` would already be an [opaque filtered response](https://fetch.spec.whatwg.org/#concept-filtered-response-opaque) with a status of 0.

Reply:
I think there might be an existing problem here in that an opaque response wouldn't expose body info.

Reply:
We set the body info to an empty body info if TAO fails.